website/content/docs/platform/k8s/helm/configuration.mdx

```diff
@@ -112,7 +113,27 @@ and consider if they're appropriate for your deployment.
 
   - `revokeOnShutdown` (`boolean: false`) - Configures all Vault Agent sidecars to revoke their token when shutting down.
 
-  - `resources` (`dictionary: {}`) - The resource requests and limits (CPU, memory, etc.) for each container of the server. This should be a YAML dictionary of a Kubernetes [ResourceRequirements](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/#resourcerequirements-v1-core) object. If this isn't specified, then the pods won't request any specific amount of resources, which limits the ability for Kubernetes to make efficient use of compute resources.<br /> **Setting this is highly recommended.**
+  - `securityContext` - Security context for the pod template and the injector container
+
+    - `pod` (`dictionary: {}`) - Defines the securityContext for the injector Pod, as YAML or a YAML-formatted multi-line templated string. Default if not specified:
+
+      ```yaml
+      runAsNonRoot: true
+      runAsGroup: {{ .Values.injector.gid | default 1000 }}
+      runAsUser: {{ .Values.injector.uid | default 100 }}
+      fsGroup: {{ .Values.injector.gid | default 1000 }}
+      ```
+
+    - `container` (`dictionary: {}`) - Defines the securityContext for the injector container, as YAML or a YAML-formatted multi-line templated string. Default if not specified:
+
+      ```yaml
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+          - ALL
+      ```
+
+  - `resources` (`dictionary: {}`) - The resource requests and limits (CPU, memory, etc.) for each container of the injector. This should be a YAML dictionary of a Kubernetes [ResourceRequirements](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#resourcerequirements-v1-core) object. If this isn't specified, then the pods won't request any specific amount of resources, which limits the ability for Kubernetes to make efficient use of compute resources.<br /> **Setting this is highly recommended.**
```

Comment:
The diff looks odd here, but I just changed `server -> injector` and updated the v1 core api link to 1.24.